_install-and-configure/additional-plugins/ingest-attachment-plugin.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: default
+title: Ingest-attachment plugin
+parent: Installing plugins
+nav_order: 20
+
+---
+
+# Ingest-attachment plugin
+
+The Ingest-attachment plugin enables OpenSearch to extract content and other information from files using the Apache text extraction library [Tika](https://tika.apache.org/).
+Supported document formats include PPT, PDF, RTF, ODF and many more ([Tika Supported Document Formats](https://tika.apache.org/2.9.2/formats.html)).
+
+The input field must be a base64-encoded binary.
+
+## Installing the plugin
+
+Install the `ingest-attachment` plugin using the following command:
+
+```sh
+./bin/opensearch-plugin install ingest-attachment
+```
+
+## Attachment processor options
+
+| Name | Required | Default | Description |
+| :--- | :--- | :--- | :--- |
```

Comment:
Line below: What does the hyphen (-) in the Default column signify?

Reply:
Doesn't contain a default.